documentation/eslintREADME.md

```diff
@@ -0,0 +1,18 @@
+## eslint-loader
+To use eslint-loader so that .vue files are automatically linted when saved in the dev environment:
+1. `npm install -D eslint eslint-loader`
```

Comment:
this should actually be installed as dev dependencies separately in the frontend and the backend projects. the package.json should be committed.